CLAUDE.md

```
@@ -0,0 +1,408 @@
+[//]: # (#;> DRUPAL_THEME)
+│   └── sites/default/     # Drupal site configuration
+├── tests/
+│   ├── behat/             # Behavioral tests (BDD)
+│   └── phpunit/           # Unit/integration tests
+├── docker-compose.yml     # Local development environment
+└── .env                   # Environment configuration
+```
+
+## Configuration Management
+
+### Exporting Configuration
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)